src/org/jetbrains/r/psi/stubs/classes/LibraryClassNameIndexProvider.kt

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2000-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+ */
+
+package org.jetbrains.r.psi.stubs.classes
+
+class LibraryClassNameIndexProvider {
+  companion object {
+    val RS4ClassNameIndex = RS4ClassNameIndex()
```

Comment:
you shouldn't create Indices directly since they are extension points, so their creation is managed by EP engine.

Reply:
I've rollbacked changes to static-classes for `classNameIndexes`